exersice1.js

```diff
@@ -0,0 +1,100 @@
+// You are talking to a friend who suddently looks at you 
+// and asks: "How old will you be in 2045?" Hmm you remember 
+// the year you were born and try to do some calculation:
+// Born in 1987 + 3 that's 1990. 90 - 40 is 50 + 5 thats 58. 
+// I will be 58! Wow that was painful! Let's fix that by making 
+// some code that automatically does this!
+
+let yearOfBirth = 1986;
```

Comment:
The ﻿`let` keyword is used only if you expect that the variable will be changed, as it allows reassignment. However, for a constant variable like ﻿`yearOfBirth`, you should use the ﻿`const` keyword. Here is an example of using ﻿let:
```
let name = "petr"
console.log(name) // petr
name = lucas 
console.log(name) // lucas
```